test/models/test_higher_order_gp.py

```diff
@@ -257,19 +257,10 @@ def test_posterior(self):
 
             # and finally test that sampling with no base samples is okay
             samples_3 = posterior.rsample(sample_shape=Size((5000,)))
-            sampled_variance = (samples_3.std(dim=0) ** 2).view(-1)
+            sampled_variance = samples_3.var(dim=0).view(-1)
             posterior_variance = posterior_variance.view(-1)
             self.assertLess(
                 (posterior_variance - sampled_variance).norm()
                 / posterior_variance.norm(),
                 5e-2,
             )
-
-
-# def TestFlattenedStandardize(BotorchTestCase):
```

Comment:
we should probably add an atomic test for this as well, but that doesn't need to be part of this PR